examples/navigation/index.js

```diff
@@ -0,0 +1,30 @@
+import 'rxjs';
```

Comment:
@gusvargas OH it works because you're effectively doing the same thing in `searchUsers.js`

https://github.com/redux-observable/redux-observable/pull/89/files/82ef94d41941f5d35b6bdd7a834bd13ffdc7165f#diff-98f4ddf4496646ae41dc030ac31bb6b5R1

You're importing the root `rxjs` to get `Observable`..the root `rx` imports _every_ operator and puts them on the prototype.

I would keep the `import 'rxjs';` in the index.js because it's the safest way to say "the app gets all of RxJS"